blocks.md

```diff
@@ -696,7 +696,7 @@ Key | Values | Required | Default
 
 ## Net
 
-Creates a block which displays the upload and download throughput for a network interface. Units are by default in bytes per second (kB/s, MB/s, etc), 
+Creates a block which displays the upload and download throughput for a network interface. Units are by default in bytes per second (kB/s, MB/s, etc),
```

Comment:
Can you remove these unrelated changes? (Submit another PR if there's an issue with the formatting in the sections apart from the Nvidia block)